src/string.zig

```diff
@@ -397,6 +397,14 @@ pub const String = extern struct {
         return BunString__fromUTF16(bytes.ptr, bytes.len);
     }
 
+    pub fn createFormat(comptime fmt: []const u8, args: anytype) !String {
+        var sba = std.heap.stackFallback(16384, bun.default_allocator);
```

Comment:
Is `16384` a constant we can stash somewhere?

Reply:
it is 2 ^ 14 bytes (16 kibibytes). this number is chosen arbitrarily.